docs/source/delta-sharing.md

```diff
@@ -0,0 +1,215 @@
+ 
+---
+description: Learn how to perform reads on Delta Sharing tables.
+---
+
+# Read Delta Sharing Tables
+
+[Delta Sharing](https://delta.io/sharing/) is an open protocol for secure real-time exchange of large datasets, which enables 
```

Comment:
This gives information on how to read the tables in the share, but how about how the recipients can discover what shares, schemas, and tables are available? How can they tell whether those tables support certain features? Do they have to use a separate tool / library to get this information?

Reply:
good point. 
Those are available through the REST API in the protocol, but not exposed through scala APIs.

We'll have to add new wrapper for that.

Reply:
We should mention or reference discovery methods somewhere in the docs.

Reply:
added.

```
Please follow the [quick start](https://github.com/delta-io/delta-sharing?tab=readme-ov-file#quick-start) to leverage the delta sharing python connector to discover the shared tables.
```